controllers/common/types.go

```diff
@@ -63,33 +63,32 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		err = r.Recover(ctx, req, chaos)
 		if err != nil {
 			r.Log.Error(err, "failed to recover chaos")
-			return ctrl.Result{Requeue: true}, nil
+			return ctrl.Result{}, nil
```

Comment:
emm, so we won't retry if the recovery fails? I think it will cause more unrecovered chaos actions.

Reply:
I agreed. What's your opinion? @cwen0

Reply:
And could we set a field like `Status`, let user be aware of it when observer chaos result?

Reply:
yes, `requeue` is better and if `recover action` was failed, we also can block this delete action. @mahjonp There is already a status field in chaos object, but we have not used it well.

Reply:
so, this pr's handle is ok?

Reply:
updated.